decidim-admin/spec/system/admin_manages_organization_admins_spec.rb

```diff
@@ -89,6 +89,16 @@
 
         expect(page).to have_no_content(other_admin.name)
       end
+
+      it "cannot remove admin rights from self" do
+        within "tr[data-user-id=\"#{user.id}\"]" do
+          expect(page).not_to have_link("Delete", visible: :hidden)
```

Comment:
visible: :all ?

How is this test impacted if the link is visible?

Reply:
Good catch! I used the completely wrong user id for searching the specific row from the list, it's now fixed to test the currently logged in admin's row and checking that they cannot delete their admin rights, like it should've been :+1: